src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,16 +1,40 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
     # TO-DO
-    
+
+    for i in range(len(merged_arr)):
+
+        if len(arrA) > 0 and len(arrB) > 0:
+            if arrA[0] < arrB[0]:
+                merged_arr[i] = arrA[0]
+                arrA.remove(arrA[0])
+            else:
+                merged_arr[i] = arrB[0]
+                arrB.remove(arrB[0])
+        else:
+            if len(arrA) == 0:
+                merged_arr[i] = arrB[0]
```

Comment:
This works, but you might want to see if you can manage doing this function without using the built-in Python list methods